controller/api/public/testdata/proxy.conf.json

```diff
@@ -0,0 +1,40 @@
+{
+    "proxyImage": {
+      "imageName": "gcr.io/linkerd-io/proxy",
+      "pullPolicy": "IfNotPresent"
+    },
+    "proxyInitImage": {
+      "imageName": "gcr.io/linkerd-io/proxy-init",
+      "pullPolicy": "IfNotPresent"
+    },
+    "destinationApiPort": {
```

Comment:
Just a heads up that once you rebase this branch and pick up #2429, you'll need to remove this field from this fixture file.